src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -135,6 +145,16 @@ public SignalRWorker(ClientJob job)
                             }
                         }
                         break;
+                    case "echoLongRunning":
+                        while (!cts.IsCancellationRequested)
+                        {
+                            await Task.Delay(_sendDelay);
```

Comment:
Maybe we should do this differently. If the send delay isn't an even division of the job duration then 2 things can happen:
1. The driver sees a "hanging" job and kills the job
2. We never send the messages